packages/shared/lib/clients/sync.client.ts

```diff
@@ -281,17 +261,19 @@ class SyncClient {
                 scheduleId
             );
 
-            if (syncData.auto_start !== false && handle) {
-                await createActivityLogMessage({
+            if (scheduleHandle) {
+                await createActivityLogMessageAndEnd({
                     level: 'info',
-                    environment_id: nangoConnection?.environment_id as number,
-                    activity_log_id: activityLogId as number,
-                    content: `Started initial background sync ${handle?.workflowId} and data updated on a schedule ${scheduleId} at ${syncData.runs} in the task queue: ${SYNC_TASK_QUEUE}`,
+                    environment_id: nangoConnection.environment_id,
+                    activity_log_id: activityLogId,
+                    content: `Created schedule ${scheduleId} at ${syncData.runs} in the task queue: ${SYNC_TASK_QUEUE}`,
```

Comment:
I realize I originally wrote this messaging but I think it can be improved. It is kind of an internal message that gives the schedule id and the task queue -- both of which the user doesn't care about. I also wonder if now it is necessary to have both an init log message and the sync kick off activity log message if the sync is `auto_start = true`. Did you discuss this with @bastienbeurier at all?

Reply:
But I guess because of how you restructured the activityLogId to not live in temporal which I agree is an improvement you need some kind of init activity log to both create and conclude -- so I'm fine with this besides the messaging needing some updates.

Reply:
Changed ed3472d